editor/assets/stylesheets/variables.scss

```diff
@@ -35,3 +35,4 @@ $editor-font: "Noto Serif", serif;
 $editor-html-font: Menlo, Consolas, monaco, monospace;
 $editor-font-size: 16px;
 $editor-line-height: 1.8em;
+$item-spacing: 10px;
```

Comment:
It's spacing between buttons in the main toolbar, as well as padding. It may be overly generic, and 10px is a pretty round number regardless.